### DISCUSSION TITLE
[Question][Hw2] print bool type

### DISCUSSION THREAD

**Discussion by yongwoo36** [prosyslab-classroom/cs524-program-analysis#417] [Q&A]
When I try to make some print bool example such as below, it needs to execute `zext i1 %cmp to i32` instruction.
```
int a = 10, b = 20; 
print(a < b)
```

But we do not support zext instruction, so it raises unsupported error.

Can I assume there is no print bool type?

**Comment by hyunsukimsokcho** [comment]
Yes, let us assume `print` only takes int type argument.

**Comment by yongwoo36** [comment]
Thanks!

**Comment by goodtaeeun** [comment]
can we also assume that ```input``` only takes in int type inputs?

**Comment by KihongHeo** [comment]
Yes.

**Comment by goodtaeeun** [comment]
Thank you!
